asdf_astropy/converters/unit/tests/test_unit.py

```diff
@@ -18,16 +18,20 @@ def vounit_compatible(unit):
 
 
 def create_vounits():
-    return {u for u in list(units.__dict__.values()) if isinstance(u, units.UnitBase) and vounit_compatible(u)}
+    return sorted(
```

Comment:
This is unrelated but while testing this I noticed that the test order was non-deterministic.